mm2src/common/log.rs

```diff
@@ -77,8 +76,8 @@ struct Gravity {
     target_thread_id: thread::ThreadId,
     /// Log chunks received from satellite threads.
     landing: SegQueue<String>,
-    /// Keeps a portiong of a recently flushed gravity log in RAM for inspection by the unit tests.
-    tail: DuplexMutex<VecDeque<String>>,
+    /// Keeps a portion of a recently flushed gravity log in RAM for inspection by the unit tests.
+    tail: Mutex<VecDeque<String>>,
```

Comment:
I think `parking_lot::Mutex` should always be imported as `PaMutex` to avoid confusion.

Reply:
Done